ui/src/dashboards/constants/templateControlBar.js

```diff
@@ -1,16 +1,17 @@
 import _ from 'lodash'
 import calculateSize from 'calculate-size'
 
-export const minDropdownWidth = 146
-export const maxDropdownWidth = 300
+export const minDropdownWidth = 120
+export const maxDropdownWidth = 330
 export const dropdownPadding = 30
 
 const valueLength = a => _.size(a.value)
 
 export const calculateDropdownWidth = (values = []) => {
-  const longestValue = _.maxBy(values, valueLength)
+  const longest = _.maxBy(values, valueLength)
```

Comment:
the default for values is [] will this ever be the case? if so, then line 14 with throw an error because maxBy will return null.